### ISSUE TITLE
A resource can contain category tag(s)

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/teamwilder#61]
## User story

As someone wanting to find resources, I want them to be grouped in multiple ways so that they are easier to discover.

## Acceptance criteria

- [ ] A resource can have 0 or many tags

## Implementation notes & questions
- The tags might be used to group resources under headings
- The tags might be used to filter resources
- The tags might be used to pull resources into feature boxes (like regional or seasonal highlights)

**Comment by katjam** [comment]
@undividual This is a place to think about how we want to be able to group resources and for what purpose.

It will also be interesting to see if we can combine with taxonomy used for  #31 

Maybe a first step will be to get a list of the existing resources and start grouping them (with Autumn?)

**Comment by undividual** [comment]
Yes, I think Autumn needs to be involved with this particular discussion from the earliest stage.

**Comment by katjam** [comment]
Cool. Let's leave the question label on here then. I'll use that to build a list for a meeting with them
